packages/slo-stat-bot/test/slo-stat-label.ts

```diff
@@ -0,0 +1,338 @@
+        .reply(202, containsSloFile)
+        .get(
+          '/repos/testOwner/testRepo/git/blobs/1d8be9e05d65e03a5e81a1c3c1bf229dce950e25'
+        )
+        .reply(202, blob)
+        .post('/repos/testOwner/testRepo/issues/6/comments', body => {
+          snapshot(body);
+          return true;
+        })
+        .reply(404);
+
+      await probot.receive({
+        name: 'pull_request.opened',
+        payload,
+        id: 'abc123',
+      });
+      requests.done();
+    });
+
+    it('Error is logged if create check fails', async () => {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const containsSloFile = require(resolve(
+        fixturesPath,
+        'events',
+        'contains_slo_file'
+      ));
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const blob = require(resolve(fixturesPath, 'events', 'invalid_blob'));
+      const requests = nock('https://api.github.com')
+        .get('/repos/testOwner/testRepo/pulls/6/files?per_page=100')
+        .reply(202, containsSloFile)
+        .get(
+          '/repos/testOwner/testRepo/git/blobs/1d8be9e05d65e03a5e81a1c3c1bf229dce950e25'
+        )
+        .reply(202, blob)
+        .post('/repos/testOwner/testRepo/issues/6/comments', body => {
+          snapshot(body);
+          return true;
+        })
+        .reply(202)
+        .post('/repos/testOwner/testRepo/check-runs', body => {
+          snapshot(body);
+          return true;
+        })
+        .reply(404);
+
+      await probot.receive({
+        name: 'pull_request.opened',
+        payload,
+        id: 'abc123',
+      });
+      requests.done();
+    });
+
+    it('An error comment is made on PR and failure check if issue_slo_rules lint is not valid', async () => {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const containsSloFile = require(resolve(
+        fixturesPath,
+        'events',
+        'contains_slo_file'
+      ));
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const blob = require(resolve(fixturesPath, 'events', 'invalid_blob'));
+      const requests = nock('https://api.github.com')
+        .get('/repos/testOwner/testRepo/pulls/6/files?per_page=100')
+        .reply(200, containsSloFile)
+        .get(
+          '/repos/testOwner/testRepo/git/blobs/1d8be9e05d65e03a5e81a1c3c1bf229dce950e25'
+        )
+        .reply(200, blob)
+        .post('/repos/testOwner/testRepo/issues/6/comments', body => {
+          snapshot(body);
+          return true;
+        })
+        .reply(200)
+        .post('/repos/testOwner/testRepo/check-runs', body => {
+          snapshot(body);
+          return true;
+        })
+        .reply(200);
+
+      await probot.receive({
+        name: 'pull_request.opened',
+        payload,
+        id: 'abc123',
+      });
+      requests.done();
+    });
+
+    it('No comment on PR and success check if issue_slo_rules lint is valid', async () => {
```

Comment:
nit: I usually try to make these read like a sentence:

> it('leaves no comment on the PR, and sets status to success, if issue_slo_rules lint is valid')

Reply:
I will fix the test sentences in the next one! (Which is coming soon)